src/sdk/sdk.ts

```diff
@@ -168,7 +168,7 @@ export class PrimeSdk {
 
     let tx: TransactionDetailsForUserOp = null;
 
-    if (!this.singleUserOp) {
+    if (this.singleUserOp.to === '') {
```

Comment:
based on https://github.com/etherspot/etherspot-prime-sdk/blob/2e4400d69d4787c27ce96a62ee9ac75ce92e9151/src/sdk/sdk.ts#L159C9-L159C42

I think this should be checked assuming `singleUserOp` can be undefined?

```suggestion
    if (this.singleUserOp?.to === '') {
```

Reply:
it can never be undefined since the initialisation was { to: '', data: '', value: ''}